app/thirdparty/res/drawable/arrowhead_left.xml

```diff
@@ -0,0 +1,17 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
license/source of all these icons? Also here, please use material icons everywhere. For most cases there should already be something in place, you just need to find the right filename.

Otherwise, <https://fonts.google.com/icons>


![grafik](https://user-images.githubusercontent.com/6735650/194113671-6be84e3a-faa4-4ebe-9b1a-778527137bdf.png)

Reply:
OK, I will search for them and then replace

Reply:
should I use some specific settings for "Fill" Weight" and so on?

Reply:
Hm the site looks a bit different then before there was no such customization before. For sure we need size 24 (as all other icons also have _24 in filename). But for the rest of toggles you can leave it unchanged.   
But if you find a icon to look better fitting, with more weight, or whatever - you can also take more weight. I leave that up to you.